src/Components/common/BigTile/styled.js

```diff
@@ -0,0 +1,274 @@
+import styled from "styled-components";
+import { ReactComponent as Vector } from "../../../images/vector.svg";
+
+/*wszystkie styled componenty sa w tym pliku ustawione 
+chronoligcznie w takiej kolejnosci w jakiej wystepuja 
+w aplikacji(patrz index.js) */
+
+export const Container = styled.div`
+  max-width: 1368px;
+  margin: auto;
+  padding: 40px;
+  display: grid;
+  gap: 0 40px;
+  grid-template-columns: minmax(auto, 312px) 1fr;
+  box-shadow: ${({ theme }) => theme.shadow.mainShadow};
+  background-color: ${({ theme }) => theme.colors.white};
+  grid-template-rows: auto 1fr;
+
+    @media (max-width: ${({ theme }) => theme.breakpoints.large}px) {
```

Comment:
Myślę, że lepiej jest wpisać od razu do theme'u stringa z wartością w px np. large: "992px" i potem po prostu przekazać 
samo large bez dopisywania px osobno.

Reply:
sluszna uwaga, nie wiedzialem poprostu jak wy to zapiszecie, ale juz to zmieniam :)